src/imitation/scripts/analyze.py

```diff
@@ -0,0 +1,95 @@
+from collections import OrderedDict
+import os.path as osp
+from typing import Optional
+
+import pandas as pd
+from sacred.observers import FileStorageObserver
+
+from imitation.scripts.config.analyze import analysis_ex
+import imitation.util.sacred as sacred_util
+from imitation.util.sacred import dict_get_nested as get
+
+
+@analysis_ex.main
+def analyze_imitation(source_dir: str,
+                      run_name: Optional[str],
+                      skip_failed_runs: bool,
+                      csv_output_path: Optional[str],
+                      verbose: bool,
+                      ) -> pd.DataFrame:
+  """
+  Args:
+    source_dir: A directory containing Sacred FileObserver subdirectories
+      associated with the `train_adversarial` Sacred script. Behavior is
+      undefined if there are Sacred subdirectories associated with other
+      scripts.
+    run_name: If provided, then only analyze results from Sacred directories
+      associated with this run name. `run_name` is compared against the
+      "experiment.name" key in `run.json`.
+    skip_failed_runs: If True, then filter out runs where the status is FAILED.
+    csv_output_path: If provided, then save a CSV output file to this path.
+    verbose: If True, then print the dataframe.
```

Comment:
I believe Sacred prints the return value of `main` anyway -- is this necessary? (Perhaps it gets truncated by default?)

Reply:
Yes, it gets truncated by default